src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/SimpleConfigJson.cs

```diff
@@ -100,7 +101,8 @@ public static string ApplyJsonChanges(
         return rootNode.ToJsonString(new JsonSerializerOptions
         {
             WriteIndented = true,
-            TypeInfoResolver = new DefaultJsonTypeInfoResolver()
+            TypeInfoResolver = new DefaultJsonTypeInfoResolver(),
+            Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping
```

Comment:
Curious - why do we need this?

Reply:
it was writing `'` and other chars in unicode. This helps with that